### PR TITLE
Unsubscribe from Newsletters Flow

### PR DESCRIPTION
**Pull request by claudianguyen**
-Added a webpage for unsubscription, linked it to the newsletter email body

### CODE REVIEW

File: frontend/jsx/Main.jsx [hunk 1]
```diff
@@ -52,7 +53,7 @@ class Main extends Component {
                 paymentMethod: PaymentMethod.PAYPAL }, // default payment method is Paypal. Other options include Check
             // allowDonationDurationChange: true,
             payPalClientId: "",
-            willPayByCheck: false // this will be used to determine the paymentMethod above.
+            willPayByCheck: false, // this will be used to determine the paymentMethod above.
```
Comment (ShahilZ) [inline frontend/jsx/Main.jsx:56]: Extra comma?

File: frontend/jsx/Main.jsx [hunk 2]
```diff
@@ -83,6 +84,7 @@ class Main extends Component {
     handleSponsorFormClick(history) {
         let self = this;
         return (values) => {
+            console.log(values.sponsor);
```
Comment (ShahilZ) [inline frontend/jsx/Main.jsx:87]: Let's remove `console.log()` statements unless this is important

File: frontend/jsx/pages/UnsubscribePage.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+            showUnsubscribedSuccessfulMessage: false
+        };
+
+        this.handleUnsubscribeClick = this.handleUnsubscribeClick.bind(this);
+    }
+
+      /***
```
Comment (ShahilZ) [inline frontend/jsx/pages/UnsubscribePage.jsx:39]: Nit pick, I think you have an extra space here, haha

File: frontend/jsx/pages/UnsubscribePage.jsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+     * Handler that submits unsubscriptions 
+     */
+    handleUnsubscribeClick(email) {
+        let self = this;
+        // I had to add headers's Content-Type to 'text-plain' in order for the backend endpoint, unsubscribe/submit/, to accept a string as the parameter. 
+        axios.post('/unsubscribe/submit/', email, { headers: {'Content-Type' : 'text/plain'}})
+        .then(function (response) {
+            console.log(response);
```
Comment (ShahilZ) [inline frontend/jsx/pages/UnsubscribePage.jsx:47]: Remove line if possible

File: src/main/resources/application-local.properties [hunk 5]
```diff
@@ -3,7 +3,9 @@
...
 # Custom properties
-aws.s3.bucket.name.images=tucklets-public-dev/images
-aws.s3.bucket.name.newsletters=tucklets-public-dev/newsletters
-aws.s3.bucket.url.images=https://tucklets-public-dev.s3-us-west-2.amazonaws.com/images/
-aws.s3.bucket.url.newsletters=https://tucklets-public-dev.s3-us-west-2.amazonaws.com/newsletters/
+aws.s3.bucket.name.images=tucklets-images-dev
+aws.s3.bucket.name.newsletters=tucklets-newsletters-dev
+aws.s3.bucket.url.images=https://tucklets-images-dev.s3-us-west-2.amazonaws.com/
+aws.s3.bucket.url.newsletters=https://tucklets-newsletters-dev.s3-us-west-2.amazonaws.com/
+domain.url=https://localhost:8443/
```
Comment (ShahilZ) [inline src/main/resources/application-local.properties:10]: Maybe `domain.base.url`?

File: src/main/resources/application.properties [hunk 6]
```diff
@@ -48,7 +48,9 @@ server.compression.enabled: true
...
 # Custom properties
-aws.s3.bucket.name.images=tucklets-public/images
-aws.s3.bucket.name.newsletters=tucklets-public/newsletters
-aws.s3.bucket.url.images=https://tucklets-public.s3-us-west-2.amazonaws.com/images/
-aws.s3.bucket.url.newsletters=https://tucklets-public.s3-us-west-2.amazonaws.com/newsletters/
+aws.s3.bucket.name.images=tucklets-images
+aws.s3.bucket.name.newsletters=tucklets-newsletters
+aws.s3.bucket.url.images=https://tucklets-images.s3-us-west-1.amazonaws.com/
+aws.s3.bucket.url.newsletters=https://tucklets-newsletters.s3-us-west-1.amazonaws.com/
+domain.url=https://tucklets.net/
```
Comment (ShahilZ) [inline src/main/resources/application.properties:55]: Same here

**Review by ShahilZ — APPROVED** [review]

File: src/main/resources/application-local.properties (hunk 5 above)
Comment (claudianguyen) [reply src/main/resources/application-local.properties:10]: Sure! I changed it to tucklets.base.url

File: src/main/resources/application.properties (hunk 6 above)
Comment (claudianguyen) [reply src/main/resources/application.properties:55]: Shore thang sir, I changed it to tucklets.base.url :)

**Pull request merged**